src/BloomBrowserUI/bookEdit/js/synphonyApi.ts

```diff
@@ -27,13 +27,13 @@ class SynphonyApi {
       lang_data.addGrapheme(this.source.letters.split(' '));
       lang_data.addWord(this.source.moreWords.split(' '));
       lang_data.LanguageSortOrder = this.source.letters.split(' ');
+    }
```

Comment:
The brace wasn't added, it was moved up. The block starting on line 32 was moved outside the if block so the closing brace moved up. If you look at lines 36-38 you will see where it used to be.